versions/development/SPEC.md

```diff
@@ -2998,7 +2998,8 @@ Varieties of the `size` function also exist for the following compound types. Th
 ## String sub(String, String, String)
 
 Given 3 String parameters `input`, `pattern`, `replace`, this function will replace any occurrence matching `pattern` in `input` by `replace`.
-`pattern` is expected to be a [regular expression](https://en.wikipedia.org/wiki/Regular_expression). Details of regex evaluation will depend on the execution engine running the WDL.
+`pattern` is expected to be a [regular expression](https://en.wikipedia.org/wiki/Regular_expression). 
+The regular expression will be evaluated as a [POSIX  Extended Regular Expression (ERE)](https://en.wikipedia.org/wiki/Regular_expression#POSIX_basic_and_extended).
 
 Example 1:
```

Comment:
If it's not already the case can you make sure the examples are valid ERE. And if nothing exercises particulars of ERE, perhaps throw in an example of that?

Reply:
I checked the existing examples and threw in some new ones. I hope they are up to standards.